<?xml version="1.0"?>

<launch>
  <node pkg="gki_sickrd_task" type="deliver_cubes_strategy.py" name="sickrd_task" output="screen">
    <rosparam command="load" file="$(find gki_sickrd_task)/config/params.yaml" />
  </node>
</launch>